LICENSE.rst

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2020, SunPy Developers
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice, this
+  list of conditions and the following disclaimer in the documentation and/or
+  other materials provided with the distribution.
+* Neither the name of the Astropy Team nor the names of its contributors may be
```

Comment:
```suggestion
* Neither the name of the SunPy Team nor the names of its contributors may be
```

Reply:
oh this is a bug in the template.

Reply:
@Cadair - yeah, I was going to ping you about that.  It's an easy one to miss!